### ISSUE TITLE
Memory Editor thinks VRAM is byte-addressed

### ISSUE THREAD

**Issue by jeffythedragonslayer** [devinacker/bsnes-plus#332]
When the Memory Editor is viewing S-PPU VRAM, there are 16 bytes per row.  The second row is labeled "0010" (which is hex for 16) which means that this Memory Editor window is displaying the VRAM as byte-addressed.  But the SNES has word-addressed VRAM.  The "Address:" label in the bottom panel has the same problem.

![image](https://user-images.githubusercontent.com/13253146/176002016-fa83e960-94a3-49cc-bc7d-0f7762c92e81.png)

**Comment by jeffythedragonslayer** [comment]
The Address field and Selected Tile Address label in the Tile Viewer also have this problem.

**Comment by devinacker** [comment]
This has always been intentional - partly just to make VRAM addressing consistent with everything else, and also since VRAM is still read/written as bytes instead of words behind the scenes.

I think it wouldn't be a bad idea to have an option to use word addressing for VRAM instead, but I'd probably make it non-default just to avoid breaking the existing convention.

**Comment by jeffythedragonslayer** [comment]
Does that default make anyone's workflow easier?  I think keeping it byte addressed makes it harder for people developing games because the addresses they're seeing aren't what they need to set VMADDL and VMADDH to in order to read/write VRAM.

**Comment by Ramsis-SNES** [comment]
I agree with @jeffythedragonslayer. At least an option to have VRAM word addressing in bsnes-plus would make things sooooo much less confusing, and more consistent, with a reasonably normal programming/debugging workflow. :-)
